mgit/README.md

```diff
@@ -0,0 +1,87 @@
+# MGit
+
+Tool for management the multi-repository architecture.
+
+## Instalation
+
+```bash
+npm install @cksource/mgit
+```
+
+## Usage
+
+Firstly, you need to create a configuration file `mgit.json`:
+
+```json
+{
+  "packages": "packages/",
+  "dependencies": {
+    "package/path": "github/url"
+  }
+}
+```
+
+* `packages` - the directory where dependencies will be cloned,
+* `dependencies` - list of dependencies.
+
+The following dependencies are proper and can be placed into configuration file:
+
+Package will be installed in `packages/@scope/package` from branch `develop`:
+
+```json
+{
+	"@scope/package": "scopes/package#develop"
+}
+```
+
+Package will be installed in `packages/package` from branch `master`:
+
+```json
+{
+	"package": "https://github.com/github/package"
```

Comment:
You didn't check the tool :(

For the config:

```json
{
  "packages": "packages/",
  "dependencies": {
    "@ckeditor/ckeditor5-autoformat": "https://github.com/ckeditor/ckeditor5-autoformat#mgit"
  }
}
```

![image](https://cloud.githubusercontent.com/assets/2270764/21846693/9b7f5dd8-d7f8-11e6-9950-ac9c430b67a5.png)

As I said, git command does not base on the configuration parameters. The tool parses the configuration and creates custom command for git.